californium-core/src/main/java/org/eclipse/californium/core/network/stack/AbstractLayer.java

```diff
@@ -192,7 +192,7 @@ public static LogOnlyLayer getInstance() {
 
 		@Override
 		public void sendRequest(final Exchange exchange, final Request request) {
-			LOGGER.error("No lower layer set for sending request [{}]", request);
+			LOGGER.trace("No lower layer set for sending request [{}]", request);
```

Comment:
There are several similar methods near it. The log levels in these methods should be consistent.